### PR TITLE
Add new tool gene.iobio

### PR DESCRIPTION
**Pull request by JasperO98**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

We added the gene.iobio tool to Galaxy based on creating URLs. Before we wanted to add clin.iobio, but it will not be supported in the long term and gene.iobio was the part of clin.iobio that we actually wanted to have. Related PRs: https://github.com/usegalaxy-eu/temporary-tools/pull/26

### CODE REVIEW

**Review by hexylena — COMMENTED** [review]
Some linting comments. I'm not sure we can write a sensible version command here without `curl` which feels wrong.

File: tools/geneiobio/geneiobio-iframe.xml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+            <option value="GRCh37">GRCh37</option>
+        </param>
+    </inputs>
+    <outputs>
+        <data name="outfile" format="html" />
+    </outputs>
+    <tests>
+    </tests>
+    <help><![CDATA[
+    Gene.iobio provides a report of impactful variants. Gene.iobio also helps prioritize variants based on multiple integrated gene:disease association algorithms and knowledge bases.
```
[the remark this answers is not in the document]
Comment (JasperO98) [reply tools/geneiobio/geneiobio-iframe.xml:98]: But in order to use the tool you'd need to make the history public right? I would say that's quite clear, but indeed we'll need to mention it here

File: tools/geneiobio/geneiobio-iframe.xml [hunk 2]
```diff
@@ -0,0 +1,116 @@
+<tool id="gene_iobio_display_generation_iframe" name="gene.iobio visualisation (iframe test)" version="4.7.1a+galaxy0">
+	<configfiles>
```
[the remark this answers is not in the document]
Comment (JasperO98) [reply tools/geneiobio/geneiobio-iframe.xml:4]: added description

File: tools/geneiobio/geneiobio-iframe.xml [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+        <param name="refgen_version" type="select" label="Select reference genome version">
+            <option value="GRCh38" selected="true">GRCh38</option>
+            <option value="GRCh37">GRCh37</option>
+        </param>
+    </inputs>
+    <outputs>
+        <data name="outfile" format="html" />
+    </outputs>
+    <tests>
+    </tests>
```
[the remark this answers is not in the document]
Comment (JasperO98) [reply tools/geneiobio/geneiobio-iframe.xml]: added test

File: tools/geneiobio/geneiobio-iframe.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (hexylena) [reply tools/geneiobio/geneiobio-iframe.xml:144]: yes, that's true, we don't mention that somewhere, we should mention that as well.

**Comment by bgruening** [comment]
@hexylena is this ready to go?

**Review by hexylena — APPROVED** [review]
Yes, looks good, let's ship it.

**Pull request merged**